sys/net/gnrc/routing/rpl/gnrc_rpl.c

```diff
@@ -265,6 +274,18 @@ void _dao_handle_send(gnrc_rpl_dodag_t *dodag)
     }
 }
 
+uint8_t gnrc_rpl_get_instance_id(bool local)
+{
+    mutex_lock(&_inst_id_mutex);
+    if (local) {
+        mutex_unlock(&_inst_id_mutex);
+        return ((_instance_id++) | GNRC_RPL_INSTANCE_ID_MSB);
```

Comment:
I am just now realizing that this mutex_unlock makes no sense, if I do not create the instance beforehand..